packages/web-components/src/progressbar/progressbar.options.ts

```diff
@@ -0,0 +1,48 @@
+import { ValuesOf } from '@microsoft/fast-foundation';
+
+/**
+ * ProgressBarThickness Constants
+ * @public
+ */
+export const ProgressBarThickness = {
+  medium: 'medium',
+  large: 'large',
+} as const;
+
+/**
+ * Applies bar thickness to the content
+ * @public
+ */
+export type ProgressBarThickness = ValuesOf<typeof ProgressBarThickness>;
+
+/**
+ * ProgressBarShape Constants
+ * @public
+ */
+export const ProgressBarShape = {
+  rounded: 'rounded',
+  rectangular: 'rectangular',
+} as const;
+
+/**
+ * Applies bar shape to the content
+ * @public
+ */
+export type ProgressBarShape = ValuesOf<typeof ProgressBarShape>;
+
+/**
+ * ProgressBarValidationState Constants
+ * @public
+ */
+export const ProgressBarValidationState = {
+  null: null,
```

Comment:
Tagging @miroslavstastny to discuss, but curious how folks feel about enumerating null in the const object. I feel like this may be better enumerated specifically in the type or in the class itself given null isn't a state, this is just a "nullable" property.

Reply:
I updated this by removing null from the object and by following your advice in the progress-bar class. Let me know if there's more to do and I can make those changes.

Reply:
Agree, no `null` in the object.